src/tss2-fapi/api/Fapi_PcrRead.c

```diff
@@ -266,6 +274,8 @@ Fapi_PcrRead_Finish(
 
 cleanup:
     /* Cleanup any intermediate results and state stored in the context. */
+    if (r)
+        SAFE_FREE(command->pcrValue);
     SAFE_FREE(command->pcrValues);
```

Comment:
Either this if block or the line after not needed

Reply:
Two different variables are freed. I will add a comment.

Reply:
Yes, sorry, I'm blind